conda_pypi/cli/pypi.py

```diff
@@ -0,0 +1,196 @@
+
+            conda pypi install requests
+
+        Install multiple PyPI packages with specific versions::
+
+            conda pypi install "numpy>=1.20" "pandas==1.5.0"
+
+        Install packages into a specific conda environment::
+
+            conda pypi install -n myenv flask django
+
+        Install packages using only PyPI (skip configured conda channels)::
+
+            conda pypi install --override-channels fastapi
+
+        """
+    )
+    install = sub_parsers.add_parser(
+        "install",
+        help=summary,
+        description=description,
+        epilog=epilog,
+    )
+    install.add_argument(
+        "--override-channels",
+        action="store_true",
+        help="Do not search default or .condarc channels. Will search pypi.",
+    )
+    install.add_argument(
+        "packages",
+        metavar="PACKAGE",
+        nargs="*",
+        help="PyPI packages to install",
+    )
+
+    # convert subcommand
+    summary = "Build and convert PyPI package or local Python project from wheel to conda package"
+    description = summary
+    epilog = dals(
+        """
+        Examples:
+
+        Convert a PyPI package to conda format without installing::
+
+            conda pypi convert requests
+
+        Convert a local Python project to conda package::
+
+            conda pypi convert ./my-python-project
+
+        Convert a package and save to a specific output folder::
+
+            conda pypi convert --output-folder ./conda-packages numpy
+
+        Convert a package from a Git repository::
+
+            conda pypi convert https://github.com/user/repo.git
+
+        """
+    )
+
+    convert = sub_parsers.add_parser(
+        "convert",
+        help=summary,
+        description=description,
+        epilog=epilog,
+    )
+
+    convert.add_argument(
+        "--output-folder",
+        help="Folder to write output package(s)",
+        type=Path,
+        required=False,
+        default=Path.cwd() / "conda-pypi-output",
+    )
+    convert.add_argument(
+        "project_path",
+        metavar="PROJECT",
+        help="Convert named path/url as wheel converted to conda.",
+    )
+
+
+def execute(args: argparse.Namespace) -> int:
+    if args.cmd == "install":
+        return execute_install(args)
+    elif args.cmd == "convert":
+        return execute_convert(args)
+    else:
+        raise ArgumentError(f"Unknown subcommand: {args.subcmd}")
+
+
+def execute_install(args: argparse.Namespace) -> int:
+    """Execute the install subcommand."""
+    if args.prefix:
+        prefix_path = Path(args.prefix).expanduser()
+    else:
+        prefix_path = Path(context.target_prefix)
+
+    converter = convert_tree.ConvertTree(prefix_path, override_channels=args.override_channels)
+    # Convert package strings to MatchSpec objects
+    match_specs = [MatchSpec(pkg) for pkg in args.packages]
+    converter.convert_tree(match_specs)
+    return 0
```

Comment:
@travishathaway pointed out that this command does not actually install anything yet. `convert_tree` simply [ends up writing a message](https://github.com/conda-incubator/conda-pypi/blob/454db45fedbecb6cb97083b5a42d6cfac7929b8f/conda_pypi/convert_tree.py#L181-L183) on what the user should do next 🤔 

Is this out of scope for this PR, or an omission? Meaning: do we need to capture this in an issue or will this be tackled here?

Reply:
As I read it, it seemed to be out of scope for this PR.